doc/JA/syntax/00_basic.md

```diff
@@ -34,10 +34,111 @@ print!(f x, y) # NG, `print!(f(x), y)`または`print!(f(x, y))`の二通りの
 print! f(x, g y, z) # NG, `print!(x, g(y), z)`または`print!(x, g(y, z))`の二通りの解釈ができてしまう
 ```
 
+## 基本的な計算
+
+### 四則演算
+
+ergも他の言語と同様の演算子(`+`, `-`, `*`, `/`, `//`, `%`)を用いて数値計算をすることができます。
+
+では、一般的な演算子を用いた計算方法を見てみましょう。
+
+```python
+>>> 1 + 1
+2
+>>> 165 * 3367 - 55555
+500000
+>>> (165 * 3367) / 5
+111111.0
+>>> (165 * 3367) // 5
+111111
+>>> (21 * 37) % 7
+0
+```
+
+ergには厳密な型があります。しかし、型推論が行われるため、型が明示的に宣言されていなければ勝手に型が推論されます。
+
+そのため、数値型であれば異なる型同士で計算をすることもできます。
+
+Rustなどの厳密な型のある言語とは違い暗黙的に型が変換されるので気をつけましょう。
```

Comment:
I think the explanation here is different, so any better explanation would be appreciated.

Reply:
Erg’s type conversion is also strict, but has the flexibility of subtyping by trait or class.

Reply:
For example, given the operation `1.0 + 1`, the 1 on the right-hand side is cast to instance `1.0` in the superclass Float; the addition of Float and Float is defined, but the addition of Int and Float is not directly defined.